dll/win32/shell32/CCopyToMenu.cpp

```diff
@@ -192,7 +192,7 @@ HRESULT CCopyToMenu::DoRealCopy(LPCMINVOKECOMMANDINFO lpici, LPCITEMIDLIST pidl)
 
 CStringW CCopyToMenu::DoGetFileTitle()
 {
-    CStringW ret = L"(file)";
+    CStringW ret; // Empty
```

Comment:
What is the point of the comment?

Reply:
It's easier to understand the code: returns empty string in case of failure.

Reply:
Removed.